CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Change Log
+
+## [0.1.0] - 2021-01-07
+
+Initial release
```

Comment:
No need to include the initial release description if we also include what was added, granted I feel the "Added" header is unnecessary here.